app/src/main/res/layout/fragment_browser.xml

```diff
@@ -60,6 +60,12 @@
                 android:elevation="24dp"
                 android:visibility="gone" />
 
+            <org.mozilla.fenix.crashes.CrashContentView
+                android:id="@+id/crash_reporter_view"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:visibility="gone" />
```

Comment:
I think I am most uncertain about this particular change hence pinging @csadilek for a review about this direction (navigating to a fragment vs displaying the view within fragment_browser) and the AppStore usage. It does make sense to me in a way to keep things cached in the AppStore and a overlaying the crash view, but just looking for more eyes.

Reply:
To me as a user it would make more sense to have the crash status and reports available even in between app restarts, until I make a decision - whether to send the report or reopen a crash tab. I understand that for now we don't need to persist this though so the `crashed` status is availabe in BrowserState but not persisted so the crashes are also kept in memory. Used Appstore because atm it's not possible to link a crash to a tab to add this info also to BrowserState. Maybe in the future.

Regarding the View that would overlay the browser also in the future I think it would be great to have this as part of the EngineView's API, lifting the need for each GV integrator to implement something similar, maybe it could come as part of the compose refactoring.

Reply:
Yes, this looks good to me though. In the future, we may be able to report per tab, and having this API as part of the EngineView also makes sense then to me.